packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -1238,9 +1274,17 @@ export const FontIcon: React.FunctionComponent<IFontIconProps>;
 // @public (undocumented)
 export function getAllSelectedOptions(options: ISelectableOption[], selectedIndices: number[]): ISelectableOption[];
 
+// @public
+export function getAriaDescribedBy(keySequences: string[]): string;
+
 // @public (undocumented)
 export function getBackgroundShade(color: IColor, shade: Shade, isInverted?: boolean): IColor | null;
 
+// Warning: (ae-forgotten-export) The symbol "IWindowWithSegments" needs to be exported by the entry point index.d.ts

```

Comment:
Seems like this ought to be exported

Reply:
It *is exported*... 

You can find it lower down. API extractor issue.

Reply:
fixed.